scripts/helpers/file.helpers.ts

```diff
@@ -3,8 +3,17 @@ import fs from 'fs';
 import { Locale } from '../factorio-build.models';
 
 export function getJsonData(file: string): unknown {
-  const str = fs.readFileSync(file).toString();
-  return JSON.parse(str);
+  try {
+    const str = fs.readFileSync(file).toString();
+    return JSON.parse(str);
+  } catch (err) {
+    if ((err as any).code === 'ENOENT') {
```

Comment:
**issue:** I would prefer to have a try/catch around the location where `space-connection-locale.json` is loaded, instead of here, which could silently ignore many legitimate errors with other files.

Reply:
Done. Turns out there were two files not generated